crates/shim/src/mount.rs

```diff
@@ -552,6 +562,12 @@ pub fn mount_rootfs(
         None
     };
 
+    unshare(CloneFlags::CLONE_FS).unwrap();
```

Comment:
why not just clone if we need to change current dir?

Reply:
@zzzzzzzzzy9

Reply:
That's right, it's my bad. I will change this.